service/build.go

```diff
@@ -58,19 +61,29 @@ func buildFileStore(cfg *config.Config) (filestore.FileStore, error) {
 
 	sess := session.Must(session.NewSession())
 	creds := credentials.NewStaticCredentials(cfg.S3AccessKey, cfg.S3AccessSecret, "")
+
+	// need a custom transport to support not verifying SSL cert
+	customTransport := &http.Transport{
+	    TLSClientConfig: &tls.Config{InsecureSkipVerify: cfg.S3DisableSSLVerify},
+        }
+	customHTTPClient := &http.Client{Transport: customTransport}
+
 	awscli := s3.New(sess, &aws.Config{
 		Credentials:      creds,
 		Endpoint:         &cfg.S3Host,
 		Region:           &cfg.S3Region,
 		DisableSSL:       &cfg.S3DisableSSL,
+		HTTPClient:       customHTTPClient,
```

Comment:
I'm worried here that we're blowing away any specific http client customizations the aws and minio clients do. Are there any docs about that?

Reply:
Not that I could find.